examples/DynaSwing/IEC/Wind/Neplan/WPP4ACurrentSource2020UDip/WPP4ACurrentSource2020.dyd

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2023, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source suite of
+    simulation tools for power systems.
+-->
+<dyn:dynamicModelsArchitecture xmlns:dyn="http://www.rte-france.com/dynawo">
+  <dyn:blackBoxModel id="WPP4A" lib="IECWPP4ACurrentSource2020" parFile="WPP4ACurrentSource2020.par" parId="WPP4A"/>
+  <dyn:blackBoxModel id="PRefPu" lib="Step" parFile="WPP4ACurrentSource2020.par" parId="PRefPu"/>
+  <dyn:blackBoxModel id="omegaRefPu" lib="DoubleStep" parFile="WPP4ACurrentSource2020.par" parId="omegaRefPu"/>
+  <dyn:blackBoxModel id="InfBus" lib="InfiniteBusWithVariations" parFile="WPP4ACurrentSource2020.par" parId="InfiniteBus"/>
+  <dyn:blackBoxModel id="Line" lib="Line" parFile="WPP4ACurrentSource2020.par" parId="Line"/>
+  <dyn:blackBoxModel id="Line1" lib="Line" parFile="WPP4ACurrentSource2020.par" parId="Line1"/>
+  <dyn:blackBoxModel id="Line2A" lib="Line" parFile="WPP4ACurrentSource2020.par" parId="Line2A"/>
+  <dyn:blackBoxModel id="Line2B" lib="Line" parFile="WPP4ACurrentSource2020.par" parId="Line2B"/>
+  <dyn:blackBoxModel id="Line3" lib="Line" parFile="WPP4ACurrentSource2020.par" parId="Line3"/>
+  <dyn:blackBoxModel id="Tfo" lib="TransformerFixedRatio" parFile="WPP4ACurrentSource2020.par" parId="Tfo"/>
+  <dyn:blackBoxModel id="Tfo1" lib="TransformerFixedRatio" parFile="WPP4ACurrentSource2020.par" parId="Tfo1"/>
+  <dyn:blackBoxModel id="ImpedantFault" lib="VariableImpedantFault" parFile="WPP4ACurrentSource2020.par" parId="ImpedantFault"/>
```

Comment:
VariableImpedantFault.xml is missing from preassemble
Could you also add it to CMakeLists.xml and NrtModels.xml?